Model/Import/Attributes.php

```diff
@@ -2,60 +2,205 @@
     const ATTRIBUTE_PREFIX = "sinch_attr_";
 
+    /**
+     * Product page size
+     */
     const PRODUCT_PAGE_SIZE = 50;
 
-    //Stats
+    /**
+     * Stats
+     * @var int
+     */
     private $attributeCount = 0;
+
+    /**
+     * @var int
+     */
     private $optionCount = 0;
 
-    //CSV parser
+    /**
+     * CSV parser
+     * @var \Magento\Framework\File\Csv
+     */
     private $csv;
 
-    //ID, CategoryID, Name, Order
+    /**
+     * ID, CategoryID, Name, Order
+     * @var
+     */
     private $category_features;
-    //ID, CategoryFeatureID, Text, Order
+
+    /**
+     * ID, CategoryFeatureID, Text, Order
+     * @var
+     */
     private $attribute_values;
-    //ID, ProductID, RestrictedValueID
+
+    /**
+     * ID, ProductID, RestrictedValueID
+     * @var
+     */
     private $product_features;
 
-    //Attributes to produce
+    /**
+     * Attributes to produce
+     * @var array
+     */
     private $attributes = [];
-    //Sinch RV -> [Prod]
+
+    /**
+     * Sinch RV -> [Prod]
+     * @var array
+     */
     private $rvProds = [];
 
+    /**
+     * @var \Magento\Catalog\Api\ProductAttributeRepositoryInterface
+     */
     private $attributeRepository;
+
+    /**
+     * @var \Magento\Catalog\Api\ProductAttributeGroupRepositoryInterface
+     */
     private $attributeGroupRepository;
+
+    /**
+     * @var \Magento\Catalog\Api\Data\ProductAttributeInterfaceFactory
+     */
     private $attributeFactory;
+
+    /**
+     * @var \Magento\Eav\Api\Data\AttributeGroupInterfaceFactory
+     */
     private $attributeGroupFactory;
+
+    /**
+     * @var \Magento\Framework\Api\SearchCriteriaBuilder
+     */
     private $searchCriteriaBuilder;
+
+    /**
+     * @var \Magento\Catalog\Api\ProductAttributeOptionManagementInterface
+     */
     private $optionManagement;
+
+    /**
+     * @var \Magento\Eav\Api\Data\AttributeOptionInterfaceFactory
+     */
     private $optionFactory;
+
+    /**
+     * @var \Magento\Catalog\Api\AttributeSetRepositoryInterface
+     */
     private $attributeSetRepository;
+
+    /**
+     * @var \Magento\Catalog\Api\ProductAttributeManagementInterface
+     */
     private $attributeManagement;
 
+    /**
+     * @var \Magento\Framework\App\ResourceConnection
+     */
     private $resourceConn;
+
+    /**
+     * @var \Magento\Framework\App\Cache\TypeListInterface
+     */
     private $cacheType;
+
+    /**
+     * @var \Magento\Catalog\Model\ResourceModel\Product\Action
+     */
     private $massProdValues;
 
+    /**
+     * @var null
+     */
     private $attributeSetCache = null;
+
+    /**
+     * @var array
+     */
     private $attributeGroupIds = [];
 
+    /**
+     * @var \Zend\Log\Logger
+     */
     private $logger;
 
+    /**
+     * @var string
+     */
     private $mappingTable;
+
+    /**
+     * @var string
+     */
     private $cpeTable;
+
+    /**
+     * @var string
+     */
     private $filterCategoriesTable;
 
+    /**
+     * @var null
```

Comment:
Commenting that the default value of a variable is null is pointless, especially when its not the actual use of the variable